README.md

```diff
@@ -100,6 +100,13 @@ obd demo
 
 See also [Quick experience](https://en.oceanbase.com/docs/community-observer-en-10000000000829647) or [Quick Start (Simplified Chinese)](https://open.oceanbase.com/quickStart) for more details.
 
+## ☸️ Start with Kubernetes
+
+You can deploy OceanBase Database instance in kubernetes cluster with [ob-operator](https://github.com/oceanbase/ob-operator) quickly. Refer to the following documents to see details.
+
+* [Quick Start](https://oceanbase.github.io/ob-operator) for ob-operator
+* [Manual](https://en.oceanbase.com/docs/common-oceanbase-database-10000000001103674) for ob-operator (in simplified Chinese)
```

Comment:
maybe this url is not right, it's the English version not simplified Chinese, we'd better use just only the gh-page cause it's version independent.